spec/features/users_can_edit_a_budget_spec.rb

```diff
@@ -92,29 +94,30 @@
       within("##{budget.id}") do
         expect(page).to have_content("£10.00")
         expect(page).to have_content("None")
-        click_on t("default.link.edit")
       end
 
-      travel_to(Date.tomorrow) do
-        fill_in "budget[value]", with: "20"
-        click_on t("default.button.submit")
+      travel_to(Date.tomorrow)
+      authenticate!(user: user)
 
-        expect(page).to have_content(t("action.budget.update.success"))
-        within("##{budget.id}") do
-          expect(page).to have_content("£20.00")
-          expect(page).to have_link("1 revision", href: activity_budget_revisions_path(budget.parent_activity_id, budget))
-          click_on t("default.link.edit")
-        end
+      click_on t("default.link.edit")
```

Comment:
Does this need to be in `within("##{budget.id}") do` like before, or is there only one budget?